packages/bindings/src/query.rs

```diff
@@ -33,6 +34,21 @@ pub enum OsmosisQuery {
         route: Vec<Step>,
         amount: SwapAmount,
     },
+    // Returns the arithmetic TWAP given base asset and quote asset.
+    ArithmeticTwap {
+        id: u64,
+        quote_asset_denom: String,
+        base_asset_denom: String,
+        start_time: SystemTime,
```

Comment:
can go time.Time be used with SystemTime?

Reply:
A type called `SystemTime` definitely doesn't sound safe/deterministic to me...

I think cosmwasm_std has a timestamp type that we should use
https://github.com/CosmWasm/cosmwasm/blob/main/packages/std/src/timestamp.rs